### PR TITLE
EN-3709: allow single-row upsert with custom row ids which are integers

### PR DESCRIPTION
**Pull request by meganschneider**
Previously this failed with upsert-error-malformed-report-item-id
because these end up as JNumbers, not JStrings. Dates appear to be taken
care of as strings already.

### CODE REVIEW

**Comment by aerust** [comment]
lgtm

File: soda-fountain-lib/src/main/scala/com/socrata/soda/server/resources/UpsertUtils.scala [hunk 1]
```diff
@@ -103,6 +103,16 @@ object UpsertUtils {
   def writeSingleRowUpsertResponse(resourceName: ResourceName, export: Export, req: HttpRequest)
                                   (response: HttpServletResponse, report: Iterator[ReportItem]): Unit = {
     var wroteOne = false
+    def exportSingleRowUpsertResponse(rowId: String): Unit = {
+      val param = ExportParam(None, None, Seq.empty[ColumnRecordLike], None,
```
Comment (JoeNunnelley) [inline soda-fountain-lib/src/main/scala/com/socrata/soda/server/resources/UpsertUtils.scala]: can you align the parameter list with the first parameter when they span multiple lines. it helps readability. thanks

Comment (meganschneider) [inline soda-fountain-lib/src/main/scala/com/socrata/soda/server/resources/UpsertUtils.scala]: Fixed!

**Comment by JoeNunnelley** [comment]
:shipit:

**Pull request merged**